### PR TITLE
Rails api relationships diagnostic - Tony Leo

### PR DESCRIPTION
**Pull request by antleo1995**
Answered the questions. Fairly confident - going 3 comfort
and 4 clarity fist of five.

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -15,23 +15,28 @@ Place your responses inside the fenced code-blocks where indivated by comments.
   join table with references to `Movies` and `Profiles`.
 
   ```md
-    # < Your Response Here >
+    The join table would be a many to many, where a profile can have many favorites
```
Comment (benjimelito) [inline diagnostic.md:18]: Indeed, but we were looking for you to write out the tables!

File: diagnostic.md [hunk 2]
```diff
@@ -15,23 +15,28 @@ Place your responses inside the fenced code-blocks where indivated by comments.
...
 
   ```rb
   class Movie < ActiveRecord::Base
+  has_many :profiles, through :favorites
   end
   ```
 
   ```rb
   class Favorite < ActiveRecord::Base
+  belongs_to: movie
```
Comment (benjimelito) [inline diagnostic.md:38]: Should look like this:
```class Favorite < ActiveRecord::Base
  belongs_to :movie, inverse_of: :favorites
  belongs_to :profile, inverse_of: :favorites
end
```

**Comment by benjimelito** [comment]
Nice work, 5/7

File: diagnostic.md (hunk 1 above)
Comment (antleo1995) [reply diagnostic.md:18]: Generally if I see MD i think you are looking for english not code.

Comment (antleo1995) [reply diagnostic.md:18]: Might wanna make the question clear on that.

Comment (benjimelito) [reply diagnostic.md:18]: Valid point, I'll open an issue for more clarification

**Pull request closed without merging**